Preparing change log for beta release (#147)
silverstripe/developer-docs: en/06_Project_Governance/07_Fixed_dependencies.md

---

> title: Fixed depnedencies

Reviewer: ```suggestion
title: Fixed dependencies
description: Dependencies which will not be changed for the lifetime of a given major release
```

Author: Done

---

# Silverstripe CMS 5 fixed dependencies

Silverstripe CMS relies on third party packages. Silverstripe CMS projects may interact directly or indirectly with those dependencies.

> We defined a list of "fixed dependencies" for the lifetime of Silverstripe CMS 5. Those dependencies will not be upgraded to new major version within the Silverstripe CMS 5 release line.

Reviewer: ```suggestion
We've defined a list of "fixed dependencies" for the lifetime of Silverstripe CMS 5. These dependencies will not be upgraded to new major version within the Silverstripe CMS 5 release line.
```

> Dependencies not in this list, may be changed or removed altogether within the lifetime of Silverstripe CMS 5.

Reviewer: ```suggestion
Dependencies not in this list may be changed or removed altogether within the lifetime of Silverstripe CMS 5.
```

> Dependency | Version

Reviewer: ```suggestion
Dependency | Fixed major release line
```

-- | --

> composer-plugin-api | 2

Reviewer: ```suggestion
[composer-plugin-api](https://getcomposer.org/doc/articles/composer-platform-dependencies.md#plugin-package-composer-plugin-api) | 2
```
That's the closest I could find to a reference for what this pseudo package actually is
